src/Assets/Lists/members.js

```diff
@@ -13,9 +13,104 @@ const memberData = {
         },
         "Logistics": {
             membersList: [
-                
+                {
+                    fullName: "Kelsang Tsomo",
+                    position: "Logistics Director",
+                    emailLink: "kelsang.tsomo@mail.utoronto.ca",
+                    LinkedInLink: "https://www.linkedin.com/in/kelsangtsomo23",
+                },
+                {
+                    fullName: "Victor Deng",
+                    position: "Logistics Director",
+                    emailLink: "victor.deng@mail.utoronto.ca",
+                    LinkedInLink: "https://www.linkedin.com/in/weian-deng/",
+                },
+                {
+                    fullName: "Victoria Yeung",
+                    position: "Logistics Director",
+                    emailLink: "victoria.yeung@mail.utoronto.ca",
+                    LinkedInLink: "https://www.linkedin.com/in/victoria-yeung5/",
+                },
+                {
+                    fullName: "Andrew Cheung",
+                    position: "Logistics Advisor",
+                    emailLink: "abc.cheung@mail.utoronto.ca",
+                    LinkedInLink: "https://www.linkedin.com/in/andrewbradleyc/",
+                },
```

Comment:
Please remove Andrew Cheung from the logistics page, as he is an advisor and should be on the advisor page.